### ISSUE TITLE
Converge Claim and Statement

### ISSUE THREAD

**Issue by SteveLasker** [ietf-wg-scitt/draft-ietf-scitt-architecture#8]
Copying over: https://github.com/ietf-scitt/draft-birkholz-scitt-architecture/issues/34

At IETF-115, discussions involved:

- Leverage existing IETF and/or W3C definitions and terms, or at least not conflict and cause confusion with existing terms.
- Minimize the concept count

### Existing Standards Terminology

- [IETF: Remote Attestation Procedures Architecture (RATS)](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html#name-artifacts)
- [IETF: SCITT Architecture](https://ietf-scitt.github.io/draft-birkholz-scitt-architecture/draft-birkholz-scitt-architecture.html#name-terminology)
- [IETF: RFC7519](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html#RFC7519)

### Proposal

Claim conflicts with RATS and W3C

Comparison of terms across orgs as the object changes state:
![](https://hackmd.io/_uploads/SklGkMSBs.png)

Instead of transitioning from `statement` to `claim` to `transparent claim`, what if we used `statement` as the base concept:
- Statement --> Signed Statment --> Transparent Statement

**Comment by rjb4standards** [comment]
IMO, A "Transparency Service" has an operator (entity) that serves as a Gatekeeper to ensure integrity of the Registry and maintain a high availability service to ensure access to the Registry data via defined API's.

**Comment by SteveLasker** [comment]
While reviewing the architecture terminology, we are considering various terms for information being placed on SCITT. What are the terms as the submission changes state? 

The current [architecture document](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#name-architecture-overview) uses:
- Statement
- Claim
- Transparent Claim

To converge on the terminology, please provide your suggestion for each stage. By the end of the week: (March 3, 2023), we'll consolidate the options and put it up for vote.

**Comment by SteveLasker** [comment]
- Statement
- Signed Statement
- Transparent Statement

**Comment by henkbirkholz** [comment]
As a potentially more palatable alternative:

* Assertion
* Signed Statement
* Transparent Statement

**Comment by rjb4standards** [comment]
REA entries in SAG-CTR are called a "Declaration";
 a formal or explicit statement or announcement.
"they issued a declaration at the close of the talks"

**Comment by fournet** [comment]
I believe think the current architecture had **4** different notions: Statement, Claim, Signed Claim, and Transparent Claim.
As we often discussed, I think the terminology in the architecture document should clearly distinguish between the notion of opaque payloads (previously known as statements) from "things with headers" which are consistently signed, made transparent, and communicated between SCITT parties. 
My proposed revised names for these 4 notions are: 
- Document (Assertion is fine too)
- Statement (The document in an envelope with selected headers)
- Signed Statement (Adding the issuer signature)
- Transparent Statement (Adding the receipt to the Signed Statement).

**Comment by SteveLasker** [comment]
What do folks think about @fournet latest suggestion?
1. It accounts for users creating a thing (a `document`), 
2. they prepare it to be submitted (`statement`: becomes a cose payload with the surrounding information, such as a from/to on an envelope)
3. it's signed by the issuer (`signed statement`)
4. It's submitted to the transparency service and accepted as a result of any registration policies (`transparent statement`)

**Comment by kaywilliams** [comment]
Agree with Maik. I prefer statement -> signed statement. For describing to lay audiences (which I do with some frequency) there are fewer concepts which makes it easier for a) the presenter to describe and b) the audience to understand.

**Comment by henkbirkholz** [comment]
https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/commit/f73a68d938d2c20d20f321a3f3e07e7ef6a4719a hopefully addresses the core issues of most parties as a step forward to rough consensus.

> Agree with Maik. I prefer statement -> signed statement. For describing to lay audiences (which I do with some frequency) there are fewer concepts which makes it easier for a) the presenter to describe and b) the audience to understand.

Fortunately, the current proposal still uses these terms 😌

**Comment by SteveLasker** [comment]
I was going to close this as #16 converges on Statement. 
However, there seems some additional discussions are happening. I'm leaving this open to assure we're capturing the history of ideas.

**Comment by SteveLasker** [comment]
I'm deferring to 117 so we can use the current #16 terminology as a common baseline for 116 discussions and slides.

**Comment by kaywilliams** [comment]
@SteveLasker asked me to post this issue here rather than in the separate issue #20 that I had created (which has now been closed)

----------------------------
In describing to lay audiences, I have found it easier to use the term 'registered statement' rather than 'transparent statement'. It reduces the number of concepts needed for understanding. Here is the workflow description I have been using:

User creates a statement
User signs a statement
User registers a statement on a transparency registry. Prior to registration, the statement is evaluated against registration policy.
User receives a receipt of registration
User shares a registered statement with interested parties
Auditors and others query the transparency registry to get a history of registered statements

See related comment on Issue https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/16
See related comment on Issue https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/20
